scout/server/blueprints/genes/templates/genes/gene.html

```diff
@@ -41,7 +41,7 @@ <h1>
                   <li class="list-group-item">
                     HGNC ID
                     <span class="float-end">
-                      <a target="_blank" href={{ record.hgnc_link }} target="_blank"> {{ hgnc_id }}</a>
+                      <a target="_blank" href={{ record.hgnc_link }} referrerpolicy="no-referrer" rel="noopener" target="_blank"> {{ hgnc_id }}</a>
```

Comment:
I've read that people use also `rel="noopener noreferrer"`, is this form better?